docs/changelog.md

```diff
@@ -2,15 +2,22 @@
 
 ## 0.2.4
 
-* Significantly increased unit test coverage, which runs before every merge of a pull request.
+* Added [Azure CLI authentication](https://github.com/databrickslabs/terraform-provider-databricks/blob/master/docs/index.md#authenticating-with-azure-cli) to bridge the gap of local development workflows and let more people use the provider.
+* All authentication is completely [lazy-initialized](https://github.com/databrickslabs/terraform-provider-databricks/pull/270), which makes it provider overall more stable.
+* Significantly increased [unit test coverage](https://codecov.io/gh/databrickslabs/terraform-provider-databricks), which runs before every merge of a pull request.
 * Introduced constantly running integration test suite environments: azsp, azcli & awsmt
+* Numerous stability improvements for clusters, mounts, libraries, notebooks, files, authentication and TLS connectivity.
 * Added ability to mount storage without explicitly defining a cluster, though it will still launch auto-terminating `terraform-mount` cluster to perform the mount.
 * `databricks_cluster` & `databricks_job` now share significant portion of configuration wiring code, therefore increasing the stability of codebase.
+* Added support for Terraform 0.13 [local builds](https://github.com/databrickslabs/terraform-provider-databricks/pull/281) for those who develop or cannot wait for next release.
+* Added AWS IAM Policy [data helpers](https://github.com/databrickslabs/terraform-provider-databricks/pull/255) to simplify new deployments.
+* [Migrated all documentation](https://github.com/databrickslabs/terraform-provider-databricks/pull/250) to Terraform Registry format, therefore having a single always-accurate place for end-user guides.
+* Internally, codebase [has been split](https://github.com/databrickslabs/terraform-provider-databricks/pull/224) into multiple packages, which should make further contributions simpler.
 
 **Deprecations**
 * `library_*` is no longer receiving fixes and will be removed in `0.3`, please rewrite cluster & job resources to use `libraries` config block.
```

Comment:
this should be `library` configuration block.

Reply:
done

Reply:
it was also supposed to be made in #259 :P